release-notes/aws/10.0.0-rc1

```diff
@@ -0,0 +1,29 @@
+## :zap:  Giant Swarm Release 10.0.0-rc1 for AWS is now active for you! :zap:
```

Comment:
`rc1` doesn't sound like a final release. Will this be `10.1.0`?

Reply:
@xh3b4sd ? What did you want to name it? @cornelius-keller said you wanted to give it a special name?

Reply:
I do not mind the name and I am not sure about the "cr" thing.

Reply:
I guess I mixed something up there. There was a discussion between @MarcelMue  and @tuommaki  on slack regarding and as far as I understood they wanted to have it as an rc as there are a few more things to test, for example china. Final release should still be `10.0.0`.

Reply:
`10.0.0` was already shipped to a customer. The next release will be `10.1.0`.